mobile/src/layout/root-navigator.tsx

```diff
@@ -9,26 +9,59 @@ import { TabNavigationOptionsFactory } from './tab-navigation-options-factory';
 import { MainTabNavigator } from './main-tab-navigator';
 
 const navOptionsFactory = new TabNavigationOptionsFactory();
-
-export const RootNavigator = TabNavigator({
-    MainTab: {
-        screen: MainTabNavigator,
-        path: '/',
-        navigationOptions: navOptionsFactory.create('Home', 'ios-home', 'ios-home-outline')
-    },
-    Feed: {
-        screen: FeedScreen,
-        path: '/feeds',
-        navigationOptions: navOptionsFactory.create('Feeds', 'ios-pulse', 'ios-pulse-outline')
+const tabNavigatorConfig: TabNavigatorConfig = {
+    tabBarPosition: 'bottom',
+    tabBarOptions: {
+        style: {
+            height: 59,
+            backgroundColor: '#2FAFCC',
+        },
+        indicatorStyle: {
+            backgroundColor: 'transparent',
+        },
+        upperCaseLabel: false,
+        labelStyle: {
+            color: '#FFFFFF',
+            fontSize: 10,
+            lineHeight: 12,
+            textAlign: 'center',
+            marginBottom: 5,
+        },
+        showIcon: true,
+        tabStyle: {
+            width: Dimensions.get('window').width / 5
+        },
     },
-    Organization: {
-        screen: OrganizationScreen,
-        path: '/organization',
-        navigationOptions: navOptionsFactory.create('Organization', 'ios-people', 'ios-people-outline')
+    animationEnabled: false,
+};
+
+export const RootNavigator = TabNavigator(
+    {
+        MainTab: {
+            screen: MainTabNavigator,
+            path: '/',
+            navigationOptions: navOptionsFactory.create('Home', require('../../src/images/inactive-tab-bar/News.png'), require('../../src/images/inactive-tab-bar/News.png'))
+        },
+        Feed: {
+            screen: FeedScreen,
+            path: '/feeds',
+            navigationOptions: navOptionsFactory.create('People', require('../../src/images/inactive-tab-bar/People.png'), require('../../src/images/inactive-tab-bar/People.png'))
+        },
+        Organization: {
+            screen: OrganizationScreen,
```

Comment:
why organizationscreen has helpdesk as an icon, why do you have `Management2` and so on. Clean it up

Reply:
It is cause all changes now contains just icons and styles according to mockup, navigation to current screens will be fixed later.